qiskit/providers/fake_provider/__init__.py

```diff
@@ -231,7 +235,7 @@
 from .fake_qobj import FakeQobj
 
 # Base classes for fake backends
-from .fake_backend import FakeBackend
+from .fake_backend import FakeBackend, FakeBackendV2
```

Comment:
This was previously imported a few lines down, but it seems that Sphinx locates its implicit type hints by static analysis and needs the explicit import to be before the `from .backends import *` one in order for it to resolve the implied crossreference in documenting the inheritance chain of the fake backends.